test/test_common/simulated_time_system.h

```diff
@@ -118,5 +118,13 @@ class SimulatedTimeSystem : public DelegatingTestTimeSystem<SimulatedTimeSystemH
   }
 };
 
+class TestUsingSimulatedTime {
```

Comment:
Can you add a comment for this? Something along the lines of:

"Inherit from this mixin in a test fixture class to use a SimulatedTimeSystem during the test."